src/egon/data/datasets/power_plants/__init__.py

```diff
@@ -976,6 +977,156 @@ def fill_missing_bus_and_geom(gens, carrier):
 
         return gens
 
+    # Write nuclear power plants in supply.egon_power_plants
+    nuclear = pd.read_csv(
+        cfg["sources"]["mastr_nuclear"],
+        usecols=[
+            "EinheitMastrNummer",
+            "Energietraeger",
+            "Nettonennleistung",
+            "Laengengrad",
+            "Breitengrad",
+            "Gemeinde",
+            "Inbetriebnahmedatum",
+        ],
+    )
+    # drop generators installed after 2019
+    nuclear["Inbetriebnahmedatum"] = pd.to_datetime(
+        nuclear["Inbetriebnahmedatum"]
+    )
+    nuclear = nuclear[nuclear["Inbetriebnahmedatum"] < "2020-01-01"]
+
+    # rename carriers
+    nuclear.loc[
+        nuclear.Energietraeger == "Kernenergie", "Energietraeger"
+    ] = "nuclear"
+    # rename columns
+    nuclear.rename(
+        columns={
+            "EinheitMastrNummer": "gens_id",
+            "Energietraeger": "carrier",
+            "Nettonennleistung": "capacity",
+            "Gemeinde": "city",
```

Comment:
I'm not sure which functions use this entry, but "Gemeinde" means municipality and not city.

Reply:
The location is used later on. To avoid confusion, the name of the column is now "location". b968b45704409033b10dc5709605100f65ec1c5a